roles/ceph-update/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+ceph:
+  # set this variable according to ceph udev rules
+  journal_guid: 45b0969e-9b03-4f30-b4c6-b4b80ceff106
```

Comment:
No. This value is decided by ceph udev rules in `/lib/udev/rules.d/95-ceph-osd.rules`. I think ceph community will not change it easily.